src/components/Dropdown/Dropdown.module.css

```diff
@@ -56,7 +56,7 @@
   border-radius: var(--eds-border-radius-md);
   box-shadow: var(--eds-box-shadow-md);
   /* Ensures the options display over the underlying body elements. */
-  z-index: var(--eds-z-index-100); 
+  z-index: var(--eds-z-index-100);
```

Comment:
Just an FYI bootstrap has its own set of z-indexes that may clash i.e. [bootstrap 3 in LP](https://github.com/twbs/bootstrap/blob/v3.4.1/less/variables.less#L274-L280) .

Haven't come across that many difficult issues regarding yet

Reply:
I think this file just had whitespace changes in it, so the lines changed should just be that.